src/loc.ml

```diff
@@ -61,12 +61,35 @@ let of_pos (fname, lnum, cnum, enum) =
   ; stop  = { pos with pos_cnum = enum }
   }
 
-let print ppf { start; stop } =
+let file_line path n =
+  Io.with_file_in ~binary:false path
```

Comment:
This should probably be `binary:true` as `dune` files are read in binary mode. It's the lexer which accepts both kinds of line endings.

Reply:
Actually it only matters for multi-line locations, which are currently not printed

Reply:
OK! I'll change that when implementing multi line highlights.